spec/index.bs

```diff
@@ -7761,54 +7815,12 @@ GPURenderBundleEncoder includes GPURenderEncoderBase;
 ### Encoding ### {#render-bundle-encoding}
 
 <script type=idl>
-dictionary GPURenderBundleEncoderDescriptor : GPUObjectDescriptorBase {
-    required sequence<GPUTextureFormat> colorFormats;
-    GPUTextureFormat depthStencilFormat;
-    GPUSize32 sampleCount = 1;
+dictionary GPURenderBundleEncoderDescriptor : GPURenderTargetsLayout, GPUObjectDescriptorBase {
```

Comment:
IDL dictionaries do not appear to support multiple-inheritance.
https://heycam.github.io/webidl/#prod-Dictionary